README.md

```diff
@@ -175,11 +158,11 @@ Each of the above has various modules that are organized by topic: passwords,
 cookies, etc. For example, if you want to timeout users after a certain period
 of inactivity, you would look in `Authlogic::Session::Timeout`.
 
-## 2. Rails
+## 2. Rails Integration
 
-Let's walk through a typical rails setup.
+Authlogic uses the rails MVC approach too! It's super simple.
```

Comment:
I like the emphasis on MVC, but I prefer a more subdued tone of voice, something like:

> Let's walk through a typical rails setup, starting with the model layer.